lib/keystore.ex

```diff
@@ -25,6 +25,9 @@ defmodule Keystore do
     privkey = decrypt!(decoded_json["crypto"], password)
     # TODO: derive from privkey and validate with this pubkey
     pubkey = Map.fetch!(decoded_json, "pubkey") |> parse_binary!()
+    if Bls.derive_pubkey(privkey) == pubkey do
```

Comment:
This check is inverted: we want to raise when the derived pubkey is different from the specified one.